pinecone/sparse/splade/splade_quora.ipynb

```diff
@@ -0,0 +1,2438 @@
+{
```

Comment:
<p>This should link to the embedding notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>